types/src/v0/impls/header.rs

```diff
@@ -425,7 +424,7 @@ impl Header {
             fee_amount,
         } in &builder_fee
         {
-            if version < MarketplaceVersion::VERSION {
+            if version < StaticVersion::<0, 3>::version() {
```

Comment:
@ss-es @imabdulbasit was `MarketplaceVersion` replaced or moved elsewhere?

Reply:
its a type now in Versions trait

Reply:
66f8d00e912e41a7702befa902d5e26f67da8681